.pending/improvements/sdk/4507-New-Signature-g

```diff
@@ -0,0 +1,3 @@
+#4507 New `Signature` generic interface available in the top level types package.
```

Comment:
This really is a breaking change.

Reply:
Fixed, thanks. And I agree to drop `improvements` - it makes very little sense to me.